README.adoc

```diff
@@ -14,12 +14,28 @@ Please see the in source asciidoc documentation or the main documentation on the
 * http://vertx.io/docs/vertx-mysql-postgresql-client/groovy/[Groovy documentation]
 * http://vertx.io/docs/vertx-mysql-postgresql-client/ruby/[Ruby documentation]
 
-== Requirements for running test suite
+== Running test suite
+
+The test suite embeds PostgreSQL and MySQL with
+
+* ru.yandex.qatools.embed:postgresql-embedded
+* com.wix:wix-embedded-mysql
+
+You can run provide the servers instead by specifying the `host` property
```

Comment:
Change to: _You can also run your own server instead by providing the `host` property_